app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,5 @@
+/*
+ *= require_tree .
+ *= require social-share-button
+ *= require_self
+ */
```

Comment:
No, you didn't. I already told you where to put that include...

Seriously, the name of the file doesn't matter in the slightest - it just needs to be included in a CSS file that will be loaded on the relevant page(s).